README.md

```diff
@@ -167,3 +167,11 @@ client.content_fields.get(['homepage_headline', 'homepage_title'])
 ### Other
```

Comment:
Please update this file to be current + accurate based on the changes in this PR